### DISCUSSION TITLE
Download Zip

### DISCUSSION THREAD

**Discussion by getaaron** [community/community#48306] [Mobile]
On my iPhone I want to download a zip file of a repository. This is possible on the desktop website but not the mobile site or app. This is a request to add "download zip" to the mobile app.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.100.0 (91333308)
Device: iPhone 11 Pro (iOS 16.4)
</details>

**Comment by Arso8888** [comment]
h

**Comment by BrandowLucas** [comment]
+1

**Comment by Dechef274** [comment]
To download a zip file of a repository on your iPhone, since you that it works on the desktop website but not the mobile site or app, here’s a workaround you can follow:1.Use Desktop Site Mode on Safari:•Open Safari on your iPhone.•Go the website of the repository (like GitHub).•Tap on the "aA" icon in the address bar.•Select "Request Desktop Website". This will load the desktop version of the site.•Navigate to the repository you want to download.•Look for the "Code" button (usually green) and click it.•From the dropdown menu, you should see the option to "Download ZIP". Tap this to start downloading.

**Comment by xmookiwa** [comment]
> +1

**Reply by xmookiwa** [reply]
https://shop.shoptoshop.co.za/qr-code/send?code=N11H33&amount=58585&reference=9689cdc728ec452faa13fe-38N25437

**Reply by xmookiwa** [reply]
https://shop.shoptoshop.co.za/qr-code/send?code=N11H33&amount=58585&reference=9689cdc728ec452faa13fe-38N25437

**Reply by xmookiwa** [reply]
https://shop.shoptoshop.co.za/qr-code/send?code=N11H33&amount=58585&reference=9689cdc728ec452faa13fe-38N25437

**Comment by Dechef274** [comment]
e

**Comment by Will2391** [comment]
> On my iPhone I want to download a zip file of a repository. This is possible on the desktop website but not the mobile site or app. This is a request to add "download zip" to the mobile app.
> 
> <details>
> <summary>ℹ️ Additional Debug Info</summary>
> GitHub for iOS, Version 1.100.0 (91333308)
> Device: iPhone 11 Pro (iOS 16.4)
> </details>
